src/OmnichannelChatSDK.ts

```diff
@@ -2773,6 +2774,61 @@ class OmnichannelChatSDK {
             }
         }
     }
+
+    /**
+     * Get persistent chat history for authenticated users.
+     * @param requestId Optional request id.
```

Comment:
This param is not present in the function prototype. Remove the line of comment.

Reply:
done